daphne_server/src/router/aggregator.rs

```diff
@@ -107,4 +148,81 @@ mod test {
 
         assert_eq!(status, StatusCode::OK);
     }
+
+    // Check that a signature produced by Daphne-Worker will be verified properly by the Clients.
+    #[test]
+    fn rondtrip_sign_hpke_config() {
+        fn os<S: AsRef<OsStr> + ?Sized>(s: &S) -> &OsStr {
+            OsStr::new(s)
+        }
+
+        let mut signing_key_file = tempfile::NamedTempFile::new().unwrap();
+        let status = Command::new("openssl")
```

Comment:
It's not ideal to add "openssl" as a dependency for running `cargo test`.

I would probably just generate a test cert and signing key and stick them in this file. Note that It's not uncommon for unit tests to have certificates. It's a bummer that you have to update the test certs once in a while, but you could make the lifetime like 10 years or something.

An alternative to calling openssl from the shell is to use the rust bindings: https://github.com/sfackler/rust-openssl/blob/master/openssl/examples/mk_certs.rs

If you prefer to call openssl from the shell, let's make sure to document somewhwere that `openssl` is required to run tests.